nesspoll/package.json

```diff
@@ -101,16 +105,13 @@
     ],
     "globals": {
       "ts-jest": {
-        "tsConfigFile": "/Users/erikschwarz/Documents/ness-inter-poll/nesspoll/tsconfig.test.json"
+        "tsConfigFile": "/Users/Lukas.DESKTOP-2EOTA3O/Documents/GitHub/ness-inter-poll/nesspoll/tsconfig.test.json"
```

Comment:
This will be painful and conflicting for everyone. Can this path be specified as relative?